src/pages/Login/index.tsx

```diff
@@ -1,5 +1,124 @@
+import { Text } from '@chakra-ui/react';
+import styled from '@emotion/styled';
+import { useNavigate } from 'react-router-dom';
+
+import KakaoSymbol from '@/assets/kakao-symbol.svg?react';
+import Logo from '@/assets/logo.svg?react';
+import IconButton from '@/components/common/IconButton';
+import Header, { HEADER_HEIGHT } from '@/components/layouts/Header';
+import { backgroundImageList } from '@/constants/login/backgroundImageList';
+import { RouterPath } from '@/routes/path';
+
 const Login = () => {
-  return <>Login</>;
+  const isMember = false; // 추후 API 연동
+  const navigate = useNavigate();
+
+  // 랜덤 배경이미지
+  const randomIndex = Math.floor(Math.random() * backgroundImageList.length);
+  const backgroundImage = backgroundImageList[randomIndex].src;
+  const backgroundImageCreator = backgroundImageList[randomIndex].creator;
+
+  return (
+    <Wrapper backgroundImage={backgroundImage}>
+      <Header
+        leftSideChildren={
+          <IconButton icon="arrow-back" color="var(--color-white)" onClick={() => navigate(-1)} />
+        }
+        rightSideChildren={
+          <IconButton
+            icon="home"
+            color="var(--color-white)"
+            onClick={() => navigate(`${RouterPath.home}`)}
```

Comment:
아래처럼 작성하지 않은 이유가 있나요?

```suggestion
            onClick={() => navigate(RouterPath.home)}
```